hack/tas-kind-cluster.yaml

```diff
@@ -9,51 +9,99 @@ nodes:
         apiVersion: kubeadm.k8s.io/v1beta3
         scheduler:
           extraArgs:
-            v: "2"
+            v: "3"
         controllerManager:
           extraArgs:
-            v: "2"
+            v: "3"
         apiServer:
           extraArgs:
             enable-aggregator-routing: "true"
-            v: "2"
+            v: "3"
   - role: worker
```

Comment:
is there some syntax to commonize the worker config? maybe there are some patches we could declare which set the log level? fyi it is not a blocker, but would be better to avoid duplication

Reply:
Fixed